frontend/src/components/abfragevarianten/AbfragevarianteFormular.vue

```diff
@@ -262,6 +264,23 @@ export default class AbfragevarianteForm extends Mixins(
     this.$emit("input", mode);
   }
 
+  private componentKeyGeschossflaecheSobonUrsaechlich = 0;
+
+  @Prop()
+  private sobonRelevant!: UncertainBoolean;
+
+  get isSobonRelevant(): UncertainBoolean {
+    return this.sobonRelevant;
+  }
+
+  get isGeschossflaecheSobonUrsaechlichPflichtfeld(): boolean {  
+    const pflichtfeld = this.isSobonRelevant === UncertainBoolean.True &&
```

Comment:
Booleanoperatoren (hier &&) bei zeilenübergreifenden Operationen in der nächsten Zeile an den Anfang stellen.

Reply:
Das würde so aussehen:

![grafik](https://user-images.githubusercontent.com/119875041/216924750-600e6ebe-99f0-4bfb-8003-ddd124dc6e87.png)

Dieser Codestyle sollte mit dem Team abgestimmt werden.